### PR TITLE
feat: add modal footer buttons for new tab and close

### PR DESCRIPTION
**Pull request by jackd248**
Usability feature to add footer buttons for modal manual view.

### CODE REVIEW

**Comment by maikschneider** [comment]
Is opening the manual in a new tab really the primary action? Currently the backend preview URL is used, so that the TYPO3 backend is opened again. Should we maybe use the frontend url?

**Comment by jackd248** [comment]
For some editors, the manual may simply be displayed too small in the iFrame. That's why the simple idea of opening it in a new tab, whether this happens in the backend or frontend, is less relevant in my opinion. But yes, you can mentioned this button as secondary or so.

File: Classes/EventListener/ModifyButtonBarEventListener.php [hunk 1]
```diff
@@ -37,6 +37,7 @@ public function __invoke(ModifyButtonBarEvent $event): void
         /** @var PageRenderer $pageRenderer */
         $pageRenderer = GeneralUtility::makeInstance(PageRenderer::class);
         $pageRenderer->loadJavaScriptModule('@xima/xima-typo3-manual/ManualModal.js');
+        $pageRenderer->addInlineLanguageLabelFile('EXT:xima-typo3-manual/Resources/Private/Language/locallang.xlf');
```
Comment (maikschneider) [inline Classes/EventListener/ModifyButtonBarEventListener.php:40]: The extension key is `xima_typo3_manual`. Did you manage to display the labels in the modal? I don't know what's going on in my installation (Base Distribution TYPO3 12.4.13) - the labels do not appear in the global `TYPO3.lang` object.

Screenshot 2024-03-28 at 10 27 11

Comment (jackd248) [reply Classes/EventListener/ModifyButtonBarEventListener.php]: 🙈 Damn it
You're right, wrong ext key with additional quotation marks within language keys caused the problems. Fixed it.

Comment (maikschneider) [reply Classes/EventListener/ModifyButtonBarEventListener.php]: Ah, I didn't notice the quotes. However I checked the global object and couldn't find the keys after adjusting the extension name. 
Hm I'll have a look later. If you change the state order (imho close should be primary and active), you can merge. Thank you!

Comment (jackd248) [reply Classes/EventListener/ModifyButtonBarEventListener.php]: > Ah, I didn't notice the quotes. However I checked the global object and couldn't find the keys after adjusting the extension name.

It's working fine within my installation, so far. 

> Hm I'll have a look later. If you change the state order (imho close should be primary and active), you can merge. Thank you!

Check, state order changed.

**Pull request merged**